lib/secrets.rb

```diff
@@ -278,4 +280,8 @@ def find_pod_ip_v4(pod_ip)
       first.
       ip_address
   end
+
+  def vault_timeout(&block)
+    Timeout.timeout 10, &block
```

Comment:
You should be able to set open_timeout and read_timeout options on Vault::Client rather than using the timeout gem...

Reply:
in fact it is already set...
```
      open_timeout: 3,
      read_timeout: 2
```

weird. why are we getting such long calls then?